content/_index.html

```diff
@@ -9,39 +9,40 @@
 ]
 +++
 <section id="intro" class="row bar no-mb">
-	<div class="col-md-offset-1 col-md-5">
+	<div class="col-sm-offset-1 col-sm-5">
 		<h1>Welcome to ReactOS</h1>
 
 		<p class="lead">
 			Imagine running your favorite Windows applications and drivers in an open-source environment you can trust.<br>
 			That's the mission of ReactOS!
 		</p>
 
-		<div class="col-md-6 row">
-			<h4>Development</h4>
-			<ul>
-				<li><a href="/getbuilds">ReactOS Builds</a></li>
-				<li><a href="/wiki/File_Bugs">Report a Bug</a></li>
-				<li><a href="/testman">Testman</a></li>
-				<li><a href="/wiki">Wiki</a></li>
-			</ul>
-		</div>
-
-		<div class="col-md-6">
-			<h4>Get in touch</h4>
-			<ul>
-				<li><a href="https://chat.reactos.org">Chat</a></li>
-				<li><a href="/forum">Forum</a></li>
-				<li><a href="/mailing-lists">Mailing Lists</a></li>
-			</ul>
+		<div>
+			<div style="display: inline-block; vertical-align: top; margin-right: 10px">
```

Comment:
```suggestion
			<div style="display: inline-block; vertical-align: top; margin-right: 50px">
```

10px looks a bit narrow in my opinion.
50px looks much better here and still fits into one row on my mobile. Even if it doesn't, your new code properly wraps it to the next row :)

Other than that, your PR is fine and especially the footer looks much better (on desktop and mobile!)